src/utils.js

```diff
@@ -86,6 +86,54 @@ var readJSON = file => {
     }
 }
 
+// FROM stackoverflow.com/questions/4253367/how-to-escape-a-json-string-containing-newline-characters-using-javascript
+var escapeJSON = str => {
+  return str
+    .replace(/[\\]/g, '\\\\')
+    .replace(/[\"]/g, '\\\"')
+    .replace(/[\/]/g, '\\/')
+    .replace(/[\b]/g, '\\b')
+    .replace(/[\f]/g, '\\f')
+    .replace(/[\n]/g, '\\n')
+    .replace(/[\r]/g, '\\r')
+    .replace(/[\t]/g, '\\t');
+};
+
+var parseHTML = file => {
+    try {
+        const [success, data, /*tag*/] = file.load_contents(null)
+        if (success) {
+            var raw = data instanceof Uint8Array
```

Comment:
Please follow the code style and don't use `var` unless you're exporting the variable. Use `const` if possible, or `let` if you need to mutate it.

Reply:
Good to know.